src/pages/coordinator/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from "react";
+
+import { Layout } from "~/layouts/DefaultLayout";
+import { useIsAdmin } from "~/hooks/useIsAdmin";
+import { useDeployment } from "~/hooks/useDeployment";
+
+
+export default function CoodirnatorDashboard() { 
+    const isAdmin = useIsAdmin();
+    const { deployStatus, deploy } = useDeployment();
+
+    const handleDeploy = useCallback(() => deploy({ amount: 99 }), []);
```

Comment:
Amount should not be hardcoded

Reply:
of course, this part just for testing now, we should get this value from the input field on coordinator page, but thinking about adding input fields and making the coordinator page having things it should have could be another PR, which means this PR is only for deployment hook and make sure it does work.